### PR TITLE
327 revise ngsi ld batch operation tests

### PR DESCRIPTION
**Pull request by SystemsPurge**
Also removed test duplicates that were present in the test cd ld test file
May not be 100% consistent due to some occasional internal problems in Orion LD that may need to be looked at more closely

### CODE REVIEW

**Review by djs0109 — APPROVED** [review]
@SystemsPurge I merged the test cases of `test_ngsi_ld_entities.py` into` test_ngsi_ld_cb.py `. Let me know whether you have other opinions

File: tests/clients/test_ngsi_ld_entity_batch_operation.py [hunk 1]
```diff
@@ -22,54 +23,18 @@ def setUp(self) -> None:
...
         """
         Cleanup entities from test server
         """
-        entity_test_types = ["filip:object:TypeA", "filip:object:TypeB",
-                             "filip:object:TypeUpdate", "filip:object:TypeDELETE"]
-        for entity_type in entity_test_types:
-            entity_list = self.cb_client.get_entity_list(entity_type=entity_type)
-            for entity in entity_list:
-                self.cb_client.delete_entity_by_id(entity_id=entity.id)
+        entity_list = self.cb_client.get_entity_list(entity_type="filip:object:test")
```
Comment (djs0109) [inline tests/clients/test_ngsi_ld_entity_batch_operation.py:35]: @SystemsPurge, can you use a similar function as in test_ngsi_ld_cb.py?

File: tests/clients/test_ngsi_ld_entity_batch_operation.py [hunk 2]
```diff
@@ -158,74 +123,63 @@ def test_entity_operations_update(self) -> None:
...
+                                      **{'temperature': {'value': self.r.randint(20,50)}}) for i in
                       range(0, 5)]
 
         self.cb_client.entity_batch_operation(entities=entities_a, action_type=ActionTypeLD.CREATE)
 
         entities_update = [ContextLDEntity(id=f"urn:ngsi-ld:test:{str(i)}",
-                                           type=f'filip:object:TypeUpdate') for i in
+                                           type=f'filip:object:test',
+                                           **{'temperature': {'value': self.r.randint(0,20)}}) for i in
                            range(3, 6)]
```
Comment (djs0109) [inline tests/clients/test_ngsi_ld_entity_batch_operation.py:136]: Is it on purpose to have an unknown entity here? i.e. urn:ngsi-ld:test:5?

**Review by djs0109 — CHANGES_REQUESTED** [review]

File: tests/clients/test_ngsi_ld_cb.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def setUp(self) -> None:
                 'value': 20.0}
         }
         self.entity = ContextLDEntity(id='urn:ngsi-ld:my:id4', type='MyType', **self.attr)
-        self.fiware_header = FiwareLDHeader(ngsild_tenant=settings.FIWARE_SERVICE)
+        self.fiware_header = FiwareLDHeader()
```
[the remark this answers is not in the document]
Comment (SystemsPurge) [reply tests/clients/test_ngsi_ld_cb.py:42]: That was just a mistake on my part, my setup does not have a default tenant and i tend to get Errors ... i assume that that tenant exists in the test cluster

File: tests/clients/test_ngsi_ld_cb.py [hunk 4]
```diff
@@ -91,28 +91,38 @@ def test_statistics(self):
...
-                          range(1000, 2001)]
-            client.entity_batch_operation(action_type=ActionTypeLD.CREATE, entities=entities_b)
-            self.assertLessEqual(len(client.get_entity_list(limit=1)), 1)
-            self.assertLessEqual(len(client.get_entity_list(limit=999)), 999)
-            self.assertLessEqual(len(client.get_entity_list(limit=1001)), 1001)
-            self.assertLessEqual(len(client.get_entity_list(limit=2001)), 2001)
-
-            client.entity_batch_operation(action_type=ActionTypeLD.DELETE, entities=entities_a)
-            client.entity_batch_operation(action_type=ActionTypeLD.DELETE, entities=entities_b)
+        self.assertLessEqual(len(self.client.get_entity_list(limit=1)), 1)
```
[the remark this answers is not in the document]
Comment (SystemsPurge) [reply tests/clients/test_ngsi_ld_cb.py:87]: I put that on the side, in case someone has some other intentions with it. You can remove it if you think it is not needed. The re-writing part in that test had to do with it failing for certain batch sizes and thats it... beyond that, im not really certain when exactly this happens

**Review by djs0109 — APPROVED** [review]
